lib/seattleflu/db/cli/command/etl/longitudinal.py

```diff
@@ -380,38 +381,31 @@ def symptoms(document: dict) -> list:
     parent_reported_symptoms = document.get("sx_specific") or []
     rn_reported_symptoms = document.get("rn_sx") or []
 
-    symptoms = list(set(parent_reported_symptoms + rn_reported_symptoms))
+    symptoms = parent_reported_symptoms + rn_reported_symptoms
 
     symptoms_map = {
-        "ear_pain": "earPainOrDischarge",
-        "fever": "feelingFeverish",
-        "headache": "headaches",
-        "myalgia": "muscleOrBodyAches",
+        "ear_pain": ["earPainOrDischarge"],
+        "fever": ["feelingFeverish"],
+        "headache": ["headaches"],
+        "myalgia": ["muscleOrBodyAches"],
         "ndv": ["nauseaOrVomiting", "diarrhea"],
         "nvd": ["nauseaOrVomiting", "diarrhea"],
-        "runny_nose": "runnyOrStuffyNose",
-        "sore_throat": "soreThroat",
-        "wob": "increasedTroubleBreathing",
-        "cough": "cough",
-        "fatigue": "fatigue",
-        "rash": "rash",
+        "runny_nose": ["runnyOrStuffyNose"],
+        "sore_throat": ["soreThroat"],
+        "wob": ["increasedTroubleBreathing"],
+        "cough": ["cough"],
+        "fatigue": ["fatigue"],
+        "rash": ["rash"],
     }
 
-    def flatten(array):
-        for symptom in array:
-            if isinstance(symptom, list):
-                for nested_symptom in flatten(symptom):
-                    yield nested_symptom
-            else:
-                yield symptom
-
     def standardize_symptom(symptom):
         try:
-            return symptom if symptom in symptoms_map.values() else symptoms_map[symptom]
+            return symptoms_map[symptom]
```

Comment:
This conditional seemed odd and unnecessary? I also think it didn't account for the list-valued entries in the map.

Reply:
Agreed, I meant to clean that up after deciding how to handle a = a mapping.